### DISCUSSION TITLE
PTM site localization

### DISCUSSION THREAD

**Discussion by diabeginner01** [vdemichev/DiaNN#485] [Q&A]
I'm looking at some DIA phosphopeptide data and noticed something that is puzzling to understand. I have a reported peptide, such as this:

LVLEQVVTS(UniMod:21)IASVADTAEEK

DIA-NN reports that the site's PTM.Q.Value is "1" and PTM.Site.Confidence is "0" .... OK, obviously this is a very sketchy localization, but what puzzles me is why it assigns the PTM to that particular serine residue? Is this random assignment, yet there are other residues (2 other Ts, and 1 additional S) where it could have assigned it if no localizing fragments were detectable. 

Does this mean there are still some fragments that point to that particular serine residue, but the overall fragment data is ambiguous (hence the confidence of 0)? Or is this completely random assignment?

**Comment by vdemichev** [comment]
Might be that the experimental spectrum best matches the predicted one for this specific localisation.
